src/analysis/modals/ComputeModal/AzureComputeModal/AzureComputeModal.js

```diff
@@ -238,6 +238,7 @@ export const AzureComputeModalBase = ({
         h(AzureApplicationConfigurationSection),
         div({ style: { ...computeStyles.whiteBoxContainer, marginTop: '1.5rem' } }, [
           h(AzureComputeProfileSelect, {
+            disabled: doesRuntimeExist(),
```

Comment:
Unrelated bug fixed along the way. If the compute modal is showing an existing cloud environment's configuration, then the machine type should be read-only.

Reply:
Or should it?... I assumed it should since the other configuration options were disabled for existing cloud environments. But maybe updating this option is supported?

Reply:
Yes, it should be disabled. The "Update" button is disabled when a cloud environment exists.
https://github.com/DataBiosphere/terra-ui/blob/73f69996dbd9bd0bc0b25e705e64b2c89e7c7e43/src/analysis/modals/ComputeModal/AzureComputeModal/AzureComputeModal.js#L139-L144